client/state/happychat/test/actions.js

```diff
@@ -8,19 +8,17 @@ import { expect } from 'chai';
 /**
  * Internal dependencies
  */
-import { setGeoLocation } from '../actions';
-import { HAPPYCHAT_SET_GEO_LOCATION } from 'state/action-types';
+import { HAPPYCHAT_CONNECTED } from 'state/action-types';
+import { setConnected } from '../actions';
 
 describe( 'actions', () => {
-	describe( '#setGeoLocation()', () => {
+	describe( '#setConnected()', () => {
 		it( 'should return an action object', () => {
-			const action = setGeoLocation( { country_long: 'Romania' } );
+			const action = setConnected( { geo_location: { country_long: 'Romania' } } );
 
 			expect( action ).to.eql( {
-				type: HAPPYCHAT_SET_GEO_LOCATION,
-				geoLocation: {
-					country_long: 'Romania',
-				},
+				type: HAPPYCHAT_CONNECTED,
+				user: { geo_location: { country_long: 'Romania' } }
```

Comment:
Doesn't `geo_location` generate linter errors? This should probably remain camelCased as `geoLocation`

Reply:
If something needs to change for consistency then we should probably change `country_long` to `countryLong`

Reply:
I'd rather change the state shape all at once in a future PR ([work in progress](https://github.com/Automattic/wp-calypso/pull/18638)).